src/main/kotlin/app/revanced/patches/youtube/shorts/overlaybuttons/ShortsOverlayButtonsPatch.kt

```diff
@@ -0,0 +1,86 @@
+package app.revanced.patches.youtube.shorts.overlaybuttons
+
+import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.PatchException
+import app.revanced.patcher.patch.options.PatchOption.PatchExtensions.stringPatchOption
+import app.revanced.patches.youtube.utils.compatibility.Constants.COMPATIBLE_PACKAGE
+import app.revanced.patches.youtube.utils.settings.SettingsPatch
+import app.revanced.util.ResourceGroup
+import app.revanced.util.copyResources
+import app.revanced.util.patch.BaseResourcePatch
+
+@Suppress("unused")
+object ShortsOverlayButtonsPatch : BaseResourcePatch(
+    name = "Shorts overlay buttons",
+    description = "Applies the new icons to the action buttons in the Shorts player.",
+    dependencies = setOf(SettingsPatch::class),
+    compatiblePackages = COMPATIBLE_PACKAGE,
+    use = true
+) {
+    private const val DEFAULT_ICON_KEY = "TikTok"
```

Comment:
As per [comment](https://github.com/inotia00/revanced-patches/pull/55#discussion_r1623165937
), the patch option `TikTok` needs to be renamed to a more appropriate name to reduce user confusion

`Round` or `Alternative` are the candidates I was thinking of

Reply:
I guess both are fine